packages/babel-plugin-proposal-private-methods/test/fixtures/private-method/tagged-template/input.js

```diff
@@ -0,0 +1,6 @@
+class Foo {
+  #tag() {
+    return this;
```

Comment:
```suggestion
    this.#tag``;
```

This `input.js` test is needed to check how Babel is compiling tagged template literals, so we have to use a template literal in the code.